packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -344,7 +344,7 @@ export class PreferencesEditorsContainer extends DockPanel {
         if (workspacePreferences) {
             workspacePreferences.title.label = 'Workspace';
             workspacePreferences.title.caption = `Workspace Preferences: ${await this.getPreferenceEditorCaption(workspacePreferenceUri!)}`;
-            workspacePreferences.title.iconClass = 'database-icon medium-yellow file-icon';
+            workspacePreferences.title.iconClass = 'database-icon medium-yellow theia-file-icons-js file-icon';
```

Comment:
There is a problem with how we are currently computing the file-icon here.
If we change file-icon themes, this value is hardcoded.

This is something that I tried to fix in #7011